spec/models/internal_change_note.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.describe InternalChangeNote do
+  let!(:internal_change_note) { build(:internal_change_note) }
+
+  it "requires a headline" do
```

Comment:
I didn't realise we didn't have any tests for `InternalChangeNote`, period! It's great you've added a validity test for `headline`, should we also have one for `author` (and `readable_created_date`) or is that out of the scope of this PR?

Reply:
Probably out of scope for this PR, but feel free to raise a PR if you want to 😁